specification/azsadmin/resource-manager/user-subscriptions/readme.azsautogen.md

```diff
@@ -0,0 +1,93 @@
+# Subscriptions Admin
```

Comment:
this file is not needed, pls remove. The private repo does nto refer to this file